R/Study_Assess.R

```diff
@@ -104,5 +106,15 @@ Study_Assess <- function(
     lAssessments <- NULL
   }
 
+  # extract results
+  lAssessments <- purrr::imap(lAssessments, function(x, y) {
+    if (is.null(x$error)) {
+      x$result
+    } else {
+      if (!bQuiet) cli::cli_alert_warning("Error in workflow {y}: {x$error}")
```

Comment:
just making sure this if statement is intentional. in the output right now it will return NULL if the the workflow had an error, unless bQuiet is TRUE then it will output the error code for the workflow instead of NULL